src/views/Orders.vue

```diff
@@ -470,6 +474,16 @@ export default defineComponent({
         console.error(err)
         this.autoCancelDays = "";
       }
+    },
+    async getMoreOrdersJobs() {
+      await this.store.dispatch("job/fetchMoreJobs", {
+        "inputFields":{
+          "enumTypeId": "ORDER_SYS_JOB",
```

Comment:
Just pass enumTypeId from here and prepare the payload in action itself, as in all the places we have the same payload.

Reply:
Done sir